backend/core/management/commands/clear_dev_images.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+Clear all images that are uploaded when in development mode.
+
+Notes
+-----
+This script is ran in docker-compose.yaml when the backend is started up.
+"""
+
+# ATTN: This file should not be edited as it removes files from a specific local directory.
+
+import os
+
+IMAGE_DIR = os.path.join(
+    os.path.dirname(
+        os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+    ),
+    "media",
+    "images",
+)
+
+
+def main() -> None:
+    """
+    Run functionality to clear uploaded images from the test images directory.
+    """
+    ENVIRONMENT = os.environ.get("ENVIRONMENT")
+
+    if ENVIRONMENT == "development":
+        print(f"In development environment. Clearing any images in {IMAGE_DIR}.")
+        try:
+            image_files_to_delete = [
+                f
+                for f in os.listdir(IMAGE_DIR)
+                if os.path.isfile(os.path.join(IMAGE_DIR, f))
+                and (f[-3:] in ["png", "jpg"] or f[-4:] in ["jpeg"])
```

Comment:
This is really good, and can be made better if we convert the file extensions .toLower(), in case a file with upper-case extension ends up in the directory.